Source/DafnyCore/AST/Grammar/TokenNewIndentCollector.cs

```diff
@@ -412,12 +419,14 @@ private class Indentations {
       }
 
       var initialMemberIndent = declWithMembers.tok.line == 0 ? indent : indent2;
-      foreach (var member in declWithMembers.Members) {
-        if (member.tok is IncludeToken) {
+      foreach (var member in declWithMembers.PreResolveChildren) {
```

Comment:
Without this change, special fields such as "_reads" for iterators, which are being assigned the same token as the iterator itself, would cause the keyword "iterator" to be indented twice.

Reply:
Ooh, that's a tricky one.